tooling/Microsoft.VisualStudio.Mac.RazorAddin/AddinMetadata.props

```diff
@@ -0,0 +1,12 @@
+﻿<Project>
+  <PropertyGroup>
+    <VSForMacVersion>7.0</VSForMacVersion>
+    <AddinVersion Condition="'$(BuildNumber)'!=''">$(VSForMacVersion).$(BuildNumber)</AddinVersion>
+    <AddinVersion Condition="'$(BuildNumber)'==''">$(VSForMacVersion).999999</AddinVersion>
+    <AddinId>RazorAddin</AddinId>
+    <AddinNamespace>Microsoft.VisualStudio.Mac</AddinNamespace>
```

Comment:
Is this right?

Reply:
Yes, this results in the final mpack being:

`Microsoft.VisualStudio.Mac.RazorAddin.mpack`